artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/Create.java

```diff
@@ -1115,6 +1116,26 @@ private void writeEtc(String source, File etcFolder, HashMap<String, String> fil
       write("etc/" + source, new File(etcFolder, source), filters, unixTarget, force);
    }
 
+   /**
+    * Format text so that it's safe to use as an XML comment.
+    *
+    * XML comments cannot contain certain elements.  This method will strip those elements out and return text that will be valid xml.
+    *
+    * XML comments cannot have "--" inside them, and they cannot end with "-"
+    * ref: https://www.w3.org/TR/2008/REC-xml-20081126/#sec-comments
+    *
+    * @param text The text to format
+    * @return text that is safe to put into an xml comment.
+    */
+   protected String safeForXmlComment(String text) {
+
+      String newText = text.replaceAll("---", "-").replaceAll("--", "-");
```

Comment:
safeForXmlComment doesn't work with all texts i.e. `safeForXmlComment("-------")` should return `--`